legacy_app.py

```
@@ -77,7 +79,11 @@
 )
 from app.services import recipe_store
 from app.services.food_store import get_food
-from app.services.intervention_trigger_engine import build_targets_next_action
+from app.services.pro_nutrition_targets import (
```

Flake8 (7.3.0)

[error] 'app.services.pro_nutrition_targets.fallback_targets_response as _fallback_targets_response' imported but unused

(F401)